backend/game/engine/test_board.py

```diff
@@ -12,3 +13,53 @@ def test_after_game_starts():
     game = BoardGame()
     game.start_game()
     assert game.status == Status.ACTIVE
+
+
+def test_game_stops():
+    game = BoardGame()
+    game.start_game()
+    assert game.status != Status.COMPLETED
+    game.end_game()
+    assert game.status == Status.COMPLETED
+
+
+def test_is_winner_by_points():
+    game = BoardGame()
+    player = Player()
+    game.add_player(player)
+    game.start_game()
+    assert game.status == Status.ACTIVE
+    current_player: Player = game.get_next_player_turn()
+    current_player.update_victory_points_by(VICTORY_POINTS_TO_WIN)
+    game.check_if_winner(current_player)
+    assert game.status == Status.COMPLETED
+
+
+def test_is_winner_by_death():
+    game = BoardGame()
+    game.add_player(Player())
+    game.add_player(Player())
+    game.start_game()
+    assert game.status == Status.ACTIVE
+    current_player: Player = game.get_next_player_turn()
+    game.check_if_winner(current_player)
+    assert game.status == Status.ACTIVE
+    current_player.update_health_by(-current_player.current_health)
+    current_player: Player = game.get_next_player_turn()
+    game.check_if_winner(current_player)
+    assert game.status == Status.COMPLETED
+
+
+def test_get_next_player_turn_consistent_order():
+    game = BoardGame()
+    for _ in range(3):
+        game.add_player(Player())
+    game.start_game()
+    player1 = game.get_next_player_turn()
+    player2 = game.get_next_player_turn()
+    player3 = game.get_next_player_turn()
+
+    for _ in range(50):
```

Comment:
Probably a bit excessive but gets the point across

Reply:
Yeah, was definitely excessive